src/Response.php

```diff
@@ -81,8 +83,10 @@ class Response implements ResponseInterface
         428 => 'Precondition Required',
```

Comment:
421 Misdirected Request is still missing.
Ref: https://httpstatuses.com/421
Ref: https://github.com/cakephp/cakephp/pull/9530/files#diff-a57bc7407c64664db661cb46c465f363R81